.code-samples.meilisearch.yaml

```diff
@@ -0,0 +1,112 @@
+# This code-sample file is used by the MeiliSearch documentation
+# Every examples written here will be automatically fetched by
+# the documentation on build
+# You can read more on https://github.com/meilisearch/documentation/tree/master/.vuepress/code-samples
+---
```

Comment:
```suggestion
```

Reply:
I think we should put this kind of comment in code-sample file. Do you?

Reply:
Actually that's true. I didn't use it, but I think you are right!

Reply:
But there is a typo ^^

Reply:
Hahaha now we have the typo in a loooot of repositories 😄